rollup.config.js

```diff
@@ -16,7 +16,7 @@ const bannerText = `${pkg.name} v${pkg.version} by Nikita Karamov\n${pkg.homepag
 
 const plugins = [
   consts({
-    urlBuilderMap,
+    fullNetworkMap,
```

Comment:
Hmm, that ain't a good idea... So you're basically loading everything about social networks into the JS file; including the icons and the colours, that don't need to be there... That is also the reason why you had to bump the limit to 7 KB for the JS file — there is no way I can ship that.

Reply:
Ah!  I didn't understand that at all, thank you!